### DISCUSSION TITLE
How to start experimenting?

### DISCUSSION THREAD

**Discussion by toastmod** [ChefKissInc/Inferno#11] [Q&A] [answered]
Hey I'd like to tinker with this repo, but unfortunately I'm a noob to how (attempting to) load iOS into qemu even works. What all do I need (am I extracting something from an IPSW?) and where do I start (what qemu parameters are needed)?

**Comment by VisualEhrmanntraut** [comment, marked as answer]
@toastmod We don't really have a guide as we are still working on the code (see dev branch). [this](https://github.com/TrungNguyen1909/qemu-t8030/wiki) should help a bit, the parameters on the dev branch are slightly different though.
